internal/start/start.go

```diff
@@ -93,6 +92,9 @@ func run(p utils.Program) error {
 			},
 		},
 	)
+	if err != nil {
+		return err
+	}
```

Comment:
This should not fail if the network already exists. Previous versions did not remove the network in some cases (https://github.com/supabase/cli/pull/393/files#r950759922), so if someone migrates from an older version, this could fail now because the network will exist.

Reply:
Ok, I see the problem.

Not  handling the error at all may hide other bugs in the future.

We could handle the specific error "network already exist" and make it continue without blocking the user and return any other type of error so that they can be catch in development?

What do you think?